server/controllers/user.js

```diff
@@ -1,14 +1,24 @@
 var model = require("../models")
 const { User } = model; 
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const saltRounds = 10;
+const SECRET_KEY = "secretkey224400";
 
 class Candidates { 
-    static signUp(req, res) { 
-    const { email, password } = req.body
+    static async create(req, res) { 
+    const { email} = req.body
+    const password  =  bcrypt.hashSync(req.body.password, saltRounds);
     const role = 'CANDIDATE'
-    return User.create({email, password, role })
-    .then(userData => res.status(201)
-    .send({ success: true, message: 'User successfully created', userData })
-    ) 
-} 
-} 
+
+      const users = await User.create({role, email, password})
```

Comment:
Let your variable name reflect what it stands for, you are creating only one user why name it users?